integration-tests/src/test/java/org/apache/druid/tests/coordinator/duty/ITAutoCompactionTest.java

```diff
@@ -376,6 +376,11 @@ private void loadData(String indexTask) throws Exception
   {
     String taskSpec = getResourceAsString(indexTask);
     taskSpec = StringUtils.replace(taskSpec, "%%DATASOURCE%%", fullDatasourceName);
+    taskSpec = StringUtils.replace(
+        taskSpec,
+        "%%SEGMENT_AVAIL_TIMEOUT_MILLIS%%",
+        jsonMapper.writeValueAsString("0")
```

Comment:
Is it worth testing what happens when this is non zero?

Do we also want to check that the IngestionStatsAndErrorsTaskReportData reports `segmentAvailabilityConfirmed` as false now?

Reply:
ya so this relates back to your comment on compaction above. It seems that the smart thing to do is add testing since it is supported by default despite my not intentionally supporting it